### PR TITLE
Implement global error handling with sentry

### PR DESCRIPTION
**Pull request by mfellner**
See #228 
and https://github.bus.zalan.do/zappr/zappr-deploy/pull/2

### CODE REVIEW

File: server/sentry.js [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+import nconf from './nconf'
+import { logger } from '../common/debug'
+
+const debug = logger('sentry', 'debug')
+const warn = logger('sentry', 'warn')
+
+const SENTRY_PUBLIC_KEY = nconf.get('SENTRY_PUBLIC_KEY')
+const SENTRY_SECRET_KEY = nconf.get('SENTRY_SECRET_KEY')
+const SENTRY_APP_ID = nconf.get('SENTRY_APP_ID')
+const SENTRY_HOST = 'app.getsentry.com'
```
Comment (j0k3r) [inline server/sentry.js]: You need to use `sentry.io` 🤓
https://blog.sentry.io/2016/08/30/sentry-io.html

**Comment by prayerslayer** [comment]
Test comment, don't mind.

**Comment by prayerslayer** [comment]
Test comment, don't mind.

**Comment by LappleApple** [comment]
Hey @mfellner @prayerslayer @spaudanjo Hey all, plans to resolve the conflicts on this PR?

**Pull request closed without merging**